### PR TITLE
service#Start, service#Stop signatures were changed

### PR DESCRIPTION
**Pull request by melekes**
See tendermint/tmlibs#45

### CODE REVIEW

File: rpc/lib/client/ws_client.go [hunk 1]
```diff
@@ -168,12 +168,14 @@ func (c *WSClient) OnStop() {}
...
 // channel is closed.
-func (c *WSClient) Stop() bool {
-	success := c.BaseService.Stop()
-	// only close user-facing channels when we can't write to them
-	c.wg.Wait()
-	close(c.ResponsesCh)
-	return success
+func (c *WSClient) Stop() error {
+	err := c.BaseService.Stop()
+	if err != nil {
```
Comment (ebuchman) [inline rpc/lib/client/ws_client.go]: should . this be `==nil` ?

Comment (melekes) [reply rpc/lib/client/ws_client.go]: right, thanks

**Comment by melekes** [comment]
```
blockchain/pool.go:188::warning: Errors unhandled.,LOW,HIGH (gas)
blockchain/pool.go:522::warning: Errors unhandled.,LOW,HIGH (gas)
blockchain/pool.go:533::warning: Errors unhandled.,LOW,HIGH (gas)
node/node.go:375::warning: Errors unhandled.,LOW,HIGH (gas)
node/node.go:384::warning: Errors unhandled.,LOW,HIGH (gas)
node/node.go:391::warning: Errors unhandled.,LOW,HIGH (gas)
```

**Comment by ebuchman** [comment]
lets just disable the `gas` checks for now

**Pull request merged**